osf/models/institution.py

```diff
@@ -18,18 +18,37 @@ class Institution(Loggable, base.ObjectIDMixin, base.BaseModel):
     # /TODO DELETE ME POST MIGRATION
 
     # TODO Remove null=True for things that shouldn't be nullable
-    banner_name = models.CharField(max_length=255, null=True, blank=True)
+
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True, null=True, default='')
+
+    # TODO Could `banner_name` and `logo_name` be a FilePathField?
+    # Both banner_name and logo_name are the name of the file under the `banners/` or `shields/` folder in
+    # `website/static/img/institutions/`. They cannot be null or empty, otherwise OSF dashboard page and institution
+    # project page won't finish loading.
+    banner_name = models.CharField(max_length=255, blank=True, default='')
```

Comment:
During the migrations all fields **MUST** be `null=True`. If you want this to change after the migrations are finished you need to add a TODO that indicates that the `null=True` should be removed.

Also, `blank=True` implies `default=''`. It will not set it at the database level but django will set values left blank to `''` during the model save.